_includes/pbc/all/install-features/202311.0/unified-commerce/install-the-warehouse-picking-feature.md

```diff
@@ -550,7 +660,117 @@ class PickingListDependencyProvider extends SprykerPickingListDependencyProvider
 }
 ```
 
-3. To enable the Backend API, register these plugins:
+3. Optional: If you've installed the OAuth Authorization, enable the following plugins:
+
+| PLUGIN                                                       | SPECIFICATION                                                                                      | PREREQUISITES | NAMESPACE                                                                |
+|--------------------------------------------------------------|----------------------------------------------------------------------------------------------------|---------------|--------------------------------------------------------------------------|
+| AuthorizeResource                                            | Registers the OAuth authorize resource.                                                            |               | SprykerEco\Glue\AuthorizationPickingAppBackendApi\Plugin\GlueApplication |
+| UserAuthCodeOauthRequestGrantTypeConfigurationProviderPlugin | Builds `OauthGrantTypeConfigurationTransfer` from the configuration of AuthorizationCode GrantType data. |               | Spryker\Zed\OauthCodeFlow\Communication\Plugin\Oauth                     |
```

Comment:
Are `AuthorizationCode` and `GrantType` separate code references?

Reply:
Only the `AuthorizationCode`.